modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -578,6 +578,19 @@ private function getEntryTitle(){
 		return "Kaltura Embed Player iFrame";
 	}
 
+	function outputCompatibilityTags(){
+		if (isset($this->request) && !empty($this->request)) {
+			$flashvars = $this->request->getFlashVars();
+			if (isset($flashvars) &&
+					isset($flashvars['forceCompatMode']) &&
```

Comment:
in general "!empty" is the same as checking if it isset as well, and even if it's not set you won't get a warning from PHP on empty check on an undeclared var. AFAIK.